### DISCUSSION TITLE
Calculating the permeability of a sample slice (.raw) using pore radius

### DISCUSSION THREAD

**Discussion by NasZN11** [PMEAL/OpenPNM#2928] [Q&A]
Hello,

May I ask if the packages PoreSpy and OpenPNM can be used to calculate permeability? I am working on this using these two packages, specifically calculating permeability using pore radius distribution. If needed, should I upload my code here?

Thank you.

**Comment by Clifford-Wang** [comment]
PoreSpy and OpenPNM are indeed capable of calculating permeability

**Comment by jgostick** [comment]
Hi @NasZN11.  As @Clifford-Wang says you *can* calculate permeability with OpenPNM, but I wouldn't use it as a "prediction".  There are too many adjustable parameters.  The better approach would be to use be to use [this LBM code](https://github.com/je-santos/MPLBM-UT) to get K, then use that to confirm that your PNM is giving the correct values.  Porespy does not have an LBM solver.
